### ISSUE TITLE
:D This sounds fun.

### ISSUE THREAD

**Issue by vidia** [mcclayton/MarkovChains#2]

**Comment by mcclayton** [comment]
Yeah I'm pretty excited about it @vidia ! As it stands, I already have it set up to read in a corpus data file and have it generate sentences from it. You can specify the order (window size) of the markov chains with a variable to get more grammatically correct, but less random sentences (from a higher order markov) or less structured sentences and more random (from a first order markov). Some other applications I want are:
1. Being able to feed in an image instead of text, and have it autogenerate an image using markov chains.
2. Being able to feed in a midi file and autogenerate music.
3. Maybe a stock market application of markov chains.

**Comment by vnataraj** [comment]
@mcclayton hopefully you can find some use for this in a big data real time pipeline auto scaling dynamic IoT cloud... but the other stuff you mentioned sounds fun 👍
